src/styles/index.scss

```diff
@@ -1,3 +1,234 @@
+  align-items: center;
+  text-transform: uppercase;
+  font-weight: 700;
+  text-decoration: none;
+  font-size: 14px;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo-container {
+    display: flex;
+    align-items: center;
+  }
+
+  &__logo {
+    width: 40px;
+    height: 40px;
+  }
+
+  &__list {
+    list-style: none;
+    display: flex;
+    align-items: center;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__link {
+    margin-left: 20px;
+    position: relative;
+    text-decoration: none;
+    text-transform: uppercase;
+    align-items: center;
+    display: flex;
+    height: 60px;
+    font-family: Roboto, sans-serif;
+    font-weight: 500;
+    font-size: 12px;
+    color: $dark;
+
+    &:hover {
+      color: $blue;
+    }
+
+    &--is-active {
+      color: $blue;
+
+      &::after {
+        content: '';
+        position: absolute;
+        left: 0;
+        bottom: 0;
+        width: 100%;
+        height: 4px;
+        background-color: $blue;
+        border-radius: 8px;
+        transform-origin: bottom left;
+      }
+    }
+  }
+
+  &__list-item {
+    &:first-child .header__link {
+      margin-left: 0;
+    }
+  }
+}
+
+.product-card {
+  width: 200px;
+  border: 1px solid $light-gray;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__image {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__code {
+    font-size: 10px;
+    color: $gray;
+    margin-top: 4px;
+  }
+
+  &__title {
+    margin: 0 0 4px;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: $dark;
+  }
+
+  @for $i from 1 through 5 {
+    &__star--#{$i} {
+      background-image: url('../images/star.svg');
+
+      @include star;
+
+      &--filled {
+        background-image: url('../images/star-active.svg');
+
+        @include star;
+      }
+    }
+  }
+
+  &__reviews {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-end;
+    justify-content: space-between;
+    margin-top: 16px;
+    margin-bottom: 24px;
+  }
+
+  &__reviews-stars {
+    display: flex;
+  }
+
+  &__reviews-total {
+    font-size: 10px;
+    color: $dark;
+  }
+
+  &__price {
+    margin-bottom: 16px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+
+    &_title {
+      font-size: 12px;
+      line-height: 18px;
+      color: $gray;
+    }
+  }
+
+  &__buy-button {
+    @include button-buy;
+
+    background-color: $blue;
+    border: none;
+    color: $white;
+
+    &:hover {
+      @include button-buy;
+
+      background-color: $white;
+      border: 1px solid $blue;
+      color: $blue;
+    }
+  }
+
+  &__star:last-child {
+    margin-right: 0;
+  }
```

Comment:
The `&__star:last-child` selector is used to remove the margin from the last star. This is good, but ensure that it doesn't cause alignment issues with the rest of the stars in the layout.